src/config.js

```diff
@@ -8,8 +11,8 @@ export default {
   useDevTools: process.env.DEV_TOOLS || false,
   print: {
     labelConfig: {
-      codeFormat: { 4: 2 },
-      canSelectPath: { 4: true, 5: true }
+      codeFormat: { '4': 2 },
```

Comment:
as long as it works as it did before, I am willing to let you a say a monkey did it ;)